packages/jsii-pacmak/test/generated-code/harness.ts

```diff
@@ -231,7 +253,7 @@ function tryStat(at: string) {
   try {
     return fs.statSync(at);
   } catch (e: any) {
-    if (e.code !== os.constants.errno.ENOENT) {
+    if (e.code !== 'ENOENT') {
```

Comment:
why change from the enum to a hardcoded string?

Reply:
Because the previous code was actually incorrect (`os.constants.errno.ENOENT` is actually the numeric value of the error code, not it's name...)